pom.xml

```diff
@@ -758,15 +756,24 @@
             </build>
         </profile>
         <profile>
-            <!-- Auto-activated profile for any JDK 9 or newer -->
-            <!-- Should contain all JDK 9+ related versions and/or test setup such as illegal-access=deny -->
-            <id>jdk9+</id>
+            <!-- Auto-activated profile for JDK 17 or newer -->
+            <id>jdk17+</id>
             <activation>
-                <jdk>(9,)</jdk>
+                <jdk>[17,)</jdk>
             </activation>
             <properties>
-                <!--In the future we will want to test with illegal access denied. As of WFLY 13 Alpha, it is not yet possible-->
-                <!--Once working, set "surefire.plugin.jdk9.args" to "[double-hyphen]illegal-access=deny"-->
+                <surefire.plugin.jdk17.args>
+                    --add-exports=java.desktop/sun.awt=ALL-UNNAMED
+                    --add-opens=java.base/java.io=ALL-UNNAMED
+                    --add-opens=java.base/java.lang=ALL-UNNAMED
+                    --add-opens=java.base/java.lang.invoke=ALL-UNNAMED
+                    --add-opens=java.base/java.security=ALL-UNNAMED
+                    --add-opens=java.base/java.util=ALL-UNNAMED
+                    --add-opens=java.management/javax.management=ALL-UNNAMED
+                    --add-opens=java.naming/javax.naming=ALL-UNNAMED
+                </surefire.plugin.jdk17.args>
+                <!--In the future we will want to test with illegal access denied. -->
+                <!--Once working, set "surefire.plugin.jdk17.args" to "[double-hyphen]illegal-access=deny"-->
```

Comment:
it seems it doesn't make sense to keep such comment anymore, the default value is `deny` as of java 16 and it can't be switched back to any other value in java 17+
https://www.oracle.com/java/technologies/javase/17-relnote-issues.html
> With this change, the java launcher option --illegal-access is obsolete. If used on the command line it causes a warning message to be issued, and otherwise has no effect.

Reply:
Ah, good point, completely forgot about that.
Will remove it.